apps/snitch_core/lib/core/calculator/calculator.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Snitch.Calculator do
```

Comment:
Why is this not inside the `Snitch.Domain` namespace?

Reply:
Why should this fall under `domain`? Shouldn't, different services have different namespacing?